### PR TITLE
[Feature/player 1424] Stereoscopic support for web mobile

### PR DESCRIPTION
**Pull request by IvanSakharovskii**
Stereoscopic for mobile Web

### CODE REVIEW

**Comment by jpfranco** [comment]
@IvanSakharovskii please add a short description to the PR titles. It's hard to keep track of them by number.

**Comment by jpfranco** [comment]
@IvanSakharovskii this PR seems to have the same updates as https://github.com/ooyala/html5-skin/pull/829. Let me know in case I missed any changes, otherwise please refer to my comments on https://github.com/ooyala/html5-skin/pull/829.

File: js/controller.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
   var Html5Skin = function (mb, id) {
     this.mb = mb;
     this.id = id;
+    this.accessibilityControls = null;
     this.videoVrSource = null;
-    this.videoVr = false;
-    this.state = {
+    this.videoVr = false;    this.state = {
```
Comment (aldigun) [inline js/controller.js]: You should put this.state on a new line.

File: js/views/playingScreen.js [hunk 2]
```diff
@@ -66,18 +64,6 @@ var PlayingScreen = React.createClass({
     }
   },
 
-  handlePlayerMouseUp: function(event) {
```
Comment (pilievOoyala) [inline js/views/playingScreen.js]: Why are we removing this function? Won't this effect the main player?  I don't see it being moved anywhere?

**Comment by pilievOoyala** [comment]
LGTM.

**Comment by aldigun** [comment]
LGTM

**Pull request merged**